lib/commanded/assertions/event_assertions.ex

```diff
@@ -191,7 +191,8 @@ defmodule Commanded.Assertions.EventAssertions do
   @doc false
   def with_subscription(application, callback_fn, opts \\ [])
       when is_function(callback_fn, 1) do
-    subscription_name = UUID.uuid4()
+    uuid_generator = Commanded.Application.uuid_generator(application, true)
```

Comment:
You can maybe use only one line here? we don't need `uuid_generator` afterward

Reply:
Okay. Made it more inline